src/components/footer.astro

```diff
@@ -64,23 +64,23 @@ const gitVersion = import.meta.env.GIT_VERSION;
           <br />
           <a
             class="underline whitespace-nowrap"
-            href="https://twitter.com/europython"
+            href="https://fosstodon.org/@europython"
           >
-            twitter.com/europython <span> ↗</span>
+            fosstodon.org/@europython <span> ↗</span>
           </a>
           <br />
           <a
             class="underline whitespace-nowrap"
-            href="https://linkedin.com/company/europython"
+            href="https://bsky.app/profile/europython.eu"
```

Comment:
Would you mind moving LinkedIn above bluesky? I think currently, it has more engagement.

Reply:
Moved!